eventkit_cloud/ui/static/ui/app/components/StatusDownloadPage/DataPackTableRow.js

```diff
@@ -43,7 +46,9 @@ export class DataPackTableRow extends Component {
                 <div
                     style={styles.data}
                 >
-                    {this.props.data}
+                    <p>
```

Comment:
Using a `<p>` element here throws an error whenever `this.props.data` is a div or some other block level element. Could the necessary styling/formatting be achieved with a more appropriate tag? reference: https://www.w3.org/TR/html401/struct/text.html#h-9.3.1